src/daemon/VirtualTerminal.cpp

```diff
@@ -173,5 +173,45 @@ namespace SDDM {
 
             close(activeVtFd);
         }
+
+        void chVt(int vt) {
```

Comment:
how is this different from jumpToVt?

Reply:
I tried jumpToVt before doing the chVt, but jumpToVt freezes the display if called on logout from Xorg sessions.

jumpToVt seems to be called only at the start of Wayland sessions and does some graphics mode setting, perhaps that's the reason for the freeze.